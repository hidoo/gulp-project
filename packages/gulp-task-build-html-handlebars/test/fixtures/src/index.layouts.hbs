---
seo:
  title: "page title"
  description: "page description."
---
{{#extend 'fuga'}}
{{#content 'body'}}
this is contents.
{{this.seo.title}}
{{this.seo.description}}
{{/content}}
{{/extend}}
