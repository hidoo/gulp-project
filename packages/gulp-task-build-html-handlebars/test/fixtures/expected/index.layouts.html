<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <title>page title</title>
    <meta name="description" content="page description." />
  </head>
  <body>
    this is contents.
page title
page description.

  </body>
</html>
